src/entity/Company.java

```diff
@@ -0,0 +1,56 @@
+package entity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+public class Company {
+    private static Company company;
+    private static AtomicBoolean instanceCreated = new AtomicBoolean(false);
+    private static Lock lock = new ReentrantLock();
+    private static final int COUNT_OF_CARS = 10;
+    private List<Car> cars = new ArrayList<>();
+    private static final Logger logger = Logger.getLogger(Company.class.getName());
+
+    private Company() {
+    }
+
+    {
```

Comment:
What is the bracket?

Reply:
You have to add cars loading after _new Company()_ in separate method, e.g. prepareCars().